views/login.ejs

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><% title %></title>
+    <link rel="stylesheet" href="./css/login.css">
+</head>
+<body class="flex-center full-display">
+    <!-- form  -->
+    <div class="formContainer">
+        <div class="formBox flex-center">
```

Comment:
Could you make this a <form> element instead of a div? You don't need to set a method or action - we will figure that out later.

Reply:
Oh! OK ..